generate_release_notes.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ] || [ "$#" -ne 0 ]; then
+  echo "------------------------------------------------------------------------------"
+  echo "Usage - generate_release_notes"
+  echo ""
+  echo "Provide the list of repositories to operate upon in a file named repo_list.txt"
+  echo "-------------------------------------------------------------------------------"
+  exit 0;
+fi;
+
+repoOwner="uc-cdis"
+startDate="2019-12-30"
```

Comment:
Maybe add a TODO so we will remember to parameterize this later
OR just mention we should rely on the previous release's tag as a baseline to start gathering release notes' bullet points.

It's a minor comment, I'm approving the PR anyway.

Reply:
For now I just added this to the documentation, we need to decide if we want to switch to tags for next release. We can finalize this once we have that clarity